heudiconv/convert.py

```diff
@@ -975,9 +979,17 @@ def save_converted_files(
         is_uncombined = (
             len(set(filter(bool, channel_names))) > 1
         )  # Check for uncombined data
-        is_complex = (
-            "M" in image_types and "P" in image_types
-        )  # Determine if data are complex (magnitude + phase)
+        CPLX_PARTS = ["MAGNITUDE", "PHASE", "IMAGINARY", "REAL"]
```

Comment:
Perhaps include "M" and "P"
```suggestion
        CPLX_PARTS = ["M", "MAGNITUDE", "P", "PHASE", "IMAGINARY", "REAL"]
```